nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -926,7 +917,7 @@ in {
             (i: v: ''
               ${occ}/bin/nextcloud-occ config:system:set trusted_domains \
                 ${toString i} --value="${toString v}"
-            '') ([ cfg.hostName ] ++ cfg.config.extraTrustedDomains));
+            '') ([ cfg.hostName ] ++ cfg.extraOptions.trusted_domains));
```

Comment:
Is this still needed btw? I mean the whole `nextcloud-occ call`.

Reply:
I would like not to touch in this PR before it's getting too complicated and taking too long to review